core/controllers/topics_and_skills_dashboard.py

```diff
@@ -132,10 +132,11 @@ class NewTopicHandler(base.BaseHandler):
     def post(self):
         """Handles POST requests."""
         name = self.payload.get('name')
-
+        abbreviated_name = self.payload.get('abbreviated_name')
         topic_domain.Topic.require_valid_name(name)
```

Comment:
Add validation for the abbreviated name as well. (Also add tests to test the validation, missing keys, etc. -- make sure the logic here catches _all_ possible issues with data submitted to the URL, since we should regard it as completely untrusted.)

Reply:
Done. Added tests in to test the validation in topic_domain_test.